packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -186,6 +189,12 @@ function TokensPanel({
   const getBalance = useCallback(
     (address: string) => {
       if (address === ETH_IDENTIFIER) {
+        if (nativeCurrency.isCustom) {
```

Comment:
Did not understand this if statement - how is the address `ETH_IDENTIFIER` and yet can be a custom native currency? 🤔

Reply:
Or maybe we should rename the identifier to something more generic?

Reply:
Fair point, I've renamed it here: https://github.com/OffchainLabs/arbitrum-token-bridge/pull/1080/commits/dfacd61b10234641b54002e78462e3379e74ea16